faunadb/strconv.go

```diff
@@ -0,0 +1,96 @@
+package faunadb
+
+import (
+	"reflect"
+	"strings"
+)
+
+func printFn(fn interface{}) string {
```

Comment:
There is no need for this function if you implement it as `func (fn fnApply) String() string`.